app/viewmodels/cya/CYAYourDetailsSummaryListHelper.scala

```diff
@@ -57,7 +57,7 @@ trait CYAYourDetailsSummaryListHelper {
       createRow(
         keyText = Text(messages("cya.userType")),
         valueContent = Text(userTypeValue),
-        action = Some(ActionItem("Url", Text(messages("cya.change"))))
+        action = Some(ActionItem(controllers.routes.UserTypeController.onLoad().url, Text(messages("cya.change"))))
```

Comment:
```suggestion
        action = Some(ActionItemHelper.createChangeActionItem(
          controllers.routes.UserTypeController.onLoad().url,
          messages("cya.userType.change")
        ))
```

Reply:
Changed and updated the visually hidden message to include some context around the change action.